### PR TITLE
Update Outlaw to 10.1.5 changes

### PR DESCRIPTION
**Pull request by evanscam97**
beacon implementation of my own added
new bomb dispenser implementation not added, instead added my own.

### CODE REVIEW

**Comment by evanscam97** [comment]
Will also need the Grand Melee check removing from slice and dice finish

**Comment by framara** [comment]
I've tried to test this but after updating to this changes i get this load error
image

**Comment by rbatche** [comment]
> I've tried to test this but after updating to this changes i get this load error image

How did you update? You may have missed some files. I recommend going to your addon folder, deleting each HR folder;
- HeroCache
- HeroDBC
- HeroLib
- HeroRotation folders

then reinstall HeroRotation, HeroLib and HeroDBC for a fresh install.

** This is also a un-merged change, so the changes in this commit wont be reflected in the current release version yet.

**Comment by framara** [comment]
@rbatche i did a manual update, overriding the modified files from the PR, in this case just the outlaw.lua 

edit: just to clarify, only outlaw stops working after changing the outlaw.lua file, rest are ok

**Comment by rbatche** [comment]
There is an error in the outlaw.lua file in this pull request, possibly syntax. You'll need to use the official outlaw.lua for now until the author has resolved it

File: HeroRotation_Rogue/Outlaw.lua [hunk 1]
```diff
@@ -46,6 +46,8 @@ local I = Item.Rogue.Outlaw
 -- Create table to exclude above trinkets from On Use function
 local OnUseExcludes = {
   I.ManicGrieftorch:ID(),
+  I.BeaconToTheBeyond:ID(),
+  I.DragonfireBombDispenser:ID(),
```
Comment (framara) [inline HeroRotation_Rogue/Outlaw.lua:50]: **BeaconToTheBeyond** and **DragonfireBombDispenser** are not defined on **Item.Rogue.Outlaw** from **Rogue.lua**

are you missing this file on the PR?

File: HeroRotation_Rogue/Outlaw.lua [hunk 2]
```diff
@@ -424,6 +428,11 @@ local function CDs ()
       if I.ManicGrieftorch:IsEquippedAndReady() and Target:FilteredTimeToDie(">", 2) and not Player:StealthUp(true, true) then
         if HR.Cast(I.ManicGrieftorch, nil, Settings.Commons.TrinketDisplayStyle) then return "Manic Grieftorch"; end
       end
+      -- beacon implement of my own, ownly suggests 0.25 after a gcd, may need changing to allow for reaction time. 
+      if I.BeaconToTheBeyond:IsEquippedAndReady() and not Player:StealthUp(true, true) and ((Target:DebuffRemains(S.BetweentheEyes) >= 3 and (not S.GhostlyStrike:IsAvailable() or Target:DebuffRemains(S.GhostlyStrike) >= 3)) or EnemiesBFCount > 2)
```
Comment (framara) [inline HeroRotation_Rogue/Outlaw.lua:432]: with this block, Outlaw.lua does not run. After removing it, it works.

File: HeroRotation_Rogue/Outlaw.lua [hunk 3]
```diff
@@ -488,6 +497,13 @@ end
 local function Build ()
   -- actions.build=sepsis,target_if=max:target.time_to_die*debuff.between_the_eyes.up,if=target.time_to_die>11&debuff.between_the_eyes.up|fight_remains<11
   -- TODO: target_if
+  -- Bomb Dispenser implementation of my own (this is not the same as the new one added to the apl as mine predates it - mine is setup to work alongside beacon. its also not in CDs because i don't think it should be treated as a cd)
```
Comment (framara) [inline HeroRotation_Rogue/Outlaw.lua:500]: with this block, Outlaw.lua does not run. After removing it, it works.

File: HeroRotation_Rogue/Outlaw.lua (hunk 1 above)
Comment (framara) [reply HeroRotation_Rogue/Outlaw.lua:50]:
```
Item.Rogue.Outlaw = {
  -- Trinkets
  ManicGrieftorch         = Item(194308, {13, 14}),
  WindscarWhetstone       = Item(137486, {13, 14}),
  BeaconToTheBeyond       = Item(203963, {13, 14}),
  DragonfireBombDispenser = Item(202610, {13, 14}),
}
```

**Comment by EvanMichaels** [comment]
Closing due to 10.2, thanks

**Pull request closed without merging**